### PR TITLE
Fixed the fqdn mapping for the method name in the trx file

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
## Description
Fixed the method name field in the UnitTestDefinition node of the trx to contain the FQDN instead of the display name of the method

### CODE REVIEW

**Comment by ShreyasRmsft** [comment]
seems like this got changed by abhishekh https://github.com/microsoft/vstest/commit/4076ff0b36205425b5f59544d578e597c3d1667f#diff-0b2fdbddedb4c3fae776162d33161e9e

**Comment by singhsarab** [comment]
@ShreyasRmsft Hoping this is on your radar

File: src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestMethod.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public TestMethod(string name, string className)
         {
             Debug.Assert(!string.IsNullOrEmpty(name), "name is null");
             Debug.Assert(!string.IsNullOrEmpty(className), "className is null");
-            this.name = name;
+            this.name = name.StartsWith($"{ClassName}.") ? name.Remove(0, $"{ClassName}.".Length) : name;
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestMethod.cs]: @ShreyasRmsft ClassName will not even be set at this point.

Comment (ShreyasRmsft) [reply src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestMethod.cs]: yep, need to use small 'c'
className

**Comment by singhsarab** [comment]
@vagisha-nidhi Are you working on this ?

**Comment by vagisha-nidhi** [comment]
> @vagisha-nidhi Are you working on this ?

No. Its already done. Closing this

**Pull request closed without merging**